static/src/stylesheets/module/external/_from-content-api.scss

```diff
@@ -116,6 +116,11 @@
 .from-content-api pre {
```

Comment:
Sorry @johnduffell, saw this comment after merge. I'm talking to James now to find out.